src/exberry_int/integration_exberry.py

```diff
@@ -132,22 +132,27 @@ async def consumer_coro(ws):
             LOG.exception(f"Error in consumer coroutine: {e}")
         finally:
             LOG.info("Ending consumer coroutine...")
+            LOG.info("Raising exception...")
```

Comment:
Why don't you. move the log into the conditional, so that you can change the log message to reflect the exception message. (On the thought that maybe the `Exception` itself will never get logged.)

Reply:
In this case the `Exception` _should_ always get logged in the `try` block for the `asyncio.gather`. This specific log was a debugging holdover that should just get removed, but it may be worth having a log message in the conditionals anyway

Reply:
I removed this and will just go with the exceptions - since they're always logged in `connect()`, I think logging it here as well would be redundant